sanic/static.py

```diff
@@ -18,7 +18,7 @@
 
 def register(app, uri, file_or_directory, pattern,
              use_modified_since, use_content_range,
-             stream_large_files, name='static', host=None):
+             stream_large_files, name='static', host=None, strict_slashes=None):
```

Comment:
Add a newline here before `strict_slashes` to fix the tests: `sanic/static.py:21:80: E501 line too long (80 > 79 characters)`

Reply:
I had my editor set to 80 character line width, fixed my editor!

When reformatting, i discovered another PEP8 violation, and then a bug in the definition of what i assume is intended to be a default threshold of 1 MiB, I can file a separate PR for the default threshold if requested.